### PR TITLE
fixed the bug on entering wrong credentials

### PR DESCRIPTION
**Pull request by rehanmm**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
     }));
     app.use(express.static("public")); // to correctly send the images and css files
 
-    var running_locally = false;
+    var running_locally = true;
```
Comment (akhilchoudhary2k) [inline app.js:23]: Change it back to false, because it's just for local testing.

File: app.js [hunk 2]
```diff
@@ -180,34 +184,55 @@
         }
     });
 
-    app.post("/login", function(req, res) {
+app.post("/login", async function (req, res, next) {
+    try {
+
+
```
Comment (akhilchoudhary2k) [inline app.js:190]: correct the indentation and remove un-necessary whitespaces like these empty lines.

File: app.js [hunk 3]
```diff
@@ -180,34 +184,55 @@
...
+                        })
+                          
+                        } else {
+
                             if (req.body.username === 'admin') res.redirect("/AdminHome");
-                            else res.redirect("/UserHome");
-                        });
-                    }
-                });
+                                                        else res.redirect("/UserHome");
```
Comment (akhilchoudhary2k) [inline app.js:219]: fix the indentation pls

File: app.js [hunk 4]
```diff
@@ -180,34 +184,55 @@
...
+                        };
+                    });
+                }
+                else {
+                    res.render("login", {
```
Comment (akhilchoudhary2k) [inline app.js:224]: here if req.body.username === admin then render the `adminlogin` page
otherwise `login` page

**Review by akhilchoudhary2k — CHANGES_REQUESTED** [review]

**Comment by akhilchoudhary2k** [comment]
@rehanmm just do these minimal changes I suggested, other than this everything looks good.
Great work !!

**Comment by akhilchoudhary2k** [comment]
@rehanmm you have created so much unnecessary diff in app.js file (have a look)
and it's very difficult for me to go through the changes now.
I think you changed indentation in the whole file, instead you should have done it only with the part of code you added.

File: app.js (hunk 2 above)
Comment (akhilchoudhary2k) [reply app.js]: @rehanmm Here I meant: give one tab indentation in front of app.post(...) so that indentation scheme matches with the rest of code.

**Comment by akhilchoudhary2k** [comment]
@rehanmm EDIT: I reviewed your changes with **ignore whitespace**, so it looks good.
Anyways the whole file indentation changed one and the same thing.

**Review by akhilchoudhary2k — APPROVED** [review]

**Comment by akhilchoudhary2k** [comment]
@rehanmm Thanks for showing interest and getting your PR merged !!

There are 2 easy open issues would you like to work? since there's little or no progress by current assignees
Have a look: https://github.com/akhilchoudhary2k/MindMatch-kgp/issues/35 & https://github.com/akhilchoudhary2k/MindMatch-kgp/issues/38

**Comment by rehanmm** [comment]
> @rehanmm you have created so much unnecessary diff in app.js file (have a look) and it's very difficult for me to go through the changes now. I think you changed indentation in the whole file, instead you should have done it only with the part of code you added.

@akhilchoudhary2k  it happened because i applied shortcut for format document and  it formatted the whole document instead of selected lines .i noticed it after  request to merge

**Pull request merged**